WordPress/src/main/java/org/wordpress/android/util/config/BlazeRemoteFields.kt

```diff
@@ -15,8 +15,7 @@ const val BLAZE_NON_DISMISSABLE_HASH_DEFAULT = "step-4"
 class BlazeNonDismissableHashConfig @Inject constructor(appConfig: AppConfig) :
     RemoteConfigField<String>(
         appConfig,
-        BLAZE_NON_DISMISSABLE_HASH_REMOTE_FIELD,
-        BLAZE_NON_DISMISSABLE_HASH_DEFAULT
```

Comment:
❓ Just to confirm: should we remove the default parameters for `RemoteConfigField`? In this case but also [this](https://github.com/wordpress-mobile/WordPress-Android/pull/18189/files#diff-a74109012289f9f7a6c6c9dd3f48b21c5d446bae6291f60ba06ce02f0d82b8bbL34), [this](https://github.com/wordpress-mobile/WordPress-Android/pull/18189/files#diff-5b22ab944bd560974e9ed1c576b133be37681814e5737c20821aad3d38a983a4L14), [this](https://github.com/wordpress-mobile/WordPress-Android/pull/18189/files#diff-5b22ab944bd560974e9ed1c576b133be37681814e5737c20821aad3d38a983a4L25) and all the others.

Reply:
It was not being used at all in the class since the actual default is set by the `RemoteFieldDefaultGenerater` annotation. So it was rather confusing to pass the default twice.